src/components/VeggyHeader/HeaderCart/CartList/CartList.jsx

```diff
@@ -0,0 +1,75 @@
+import './CartList.css'
+import cartListImg from '../../../../../public/img/empty-cart.png'
+import '../HeaderCart.css'
+
+export  default function CartList ({modalActive, addProduct,setAddProduct}) {
+
+    return (
+        <>
+        {modalActive &&
+            <div className='cartListContainer'>
+                <div className='cartListRow'
+                     onClick={e => e.stopPropagation()}>
+                    <div className={addProduct.length > 0 ? 'emptyImageRow disable' : 'emptyImageRow'}>
```

Comment:
В тебе і там і там завжди буде класс emptyImageRow. Навіщо тоді його дублювати? Напиши типу 
```suggestion
                    <div className={`emptyImageRow ${addProduct.length > 0 ? "disable" : ""}`}>
```